### ISSUE TITLE
FAILED UPLOAD SOLUTION

### ISSUE THREAD

**Issue by AlejandroTuzzi** [Shakker-Labs/ComfyUI-IPAdapter-Flux#63]
Hello, I found a solution for this Issue using ChatGPT ---> (Awesome)

Issue:
The error occurred due to a typo in the function name is_model_pathched. It should have been is_model_patched.

Solution:
Search for the Typo:

Using Visual Studio Code, I searched for the term is_model_pathched across the entire directory.
The search revealed that this typo was present in the file ipadapter_flux_advanced.py.
Fix the Typo:

I replaced every instance of is_model_pathched with the correct function name is_model_patched.
Reload ComfyUI:

After making the change, I restarted ComfyUI, and the node loaded successfully!

I hope it is useful! SeeU!

**Comment by AlejandroTuzzi** [comment]
![ComfyUI demo](https://github.com/user-attachments/assets/42a16f20-2d51-4c76-8d51-2582cc4f0e11)

And, ¡It works perfect!

**Comment by al-swaiti** [comment]
thanks ,,, its work now

**Comment by al-swaiti** [comment]
https://github.com/al-swaiti/ComfyUI-IPAdapter-Flux

**Comment by AlejandroTuzzi** [comment]
> https://github.com/al-swaiti/ComfyUI-IPAdapter-Flux

Yes, Is for that. I'm testing right now and it is working well, the only problem is that the resulting image has a kind of banding or something like that... 

![MarkuryFLUX_00122_](https://github.com/user-attachments/assets/5396e0d6-d075-4c38-95aa-2f03a22c2a31)
![MarkuryFLUX_00138_](https://github.com/user-attachments/assets/8a47bbb4-4720-405d-9c59-84c924c54469)

**Comment by al-swaiti** [comment]
i think the source image must be in high quality

**Comment by al-swaiti** [comment]
nice setup ,,,, mmm try mine https://civitai.com/models/628210/advanced-ai-art-remix-workflow-for-flux

**Comment by al-swaiti** [comment]
can you share those text files 
![image](https://github.com/user-attachments/assets/4021f5eb-3dce-4184-8c43-0a2824e2c5ea)

**Comment by AlejandroTuzzi** [comment]
I'll try, thanks!!! 
Sure! Here you have.

[Poses.txt](https://github.com/user-attachments/files/18275304/Poses.txt)

**Comment by AlejandroTuzzi** [comment]
![Poses](https://github.com/user-attachments/assets/4e428a63-1fa2-4ee9-a7fb-a838057440f4)

Each line is a pose

**Comment by al-swaiti** [comment]
mmm i made some thing like this for my styler node but more advance ,,, ill add those poses 

https://github.com/user-attachments/assets/e0a671d6-5e68-41f6-9a38-5f3b45621091

**Comment by al-swaiti** [comment]
did u fixed those lines 
![image](https://github.com/user-attachments/assets/c8db7060-12da-4405-be0d-b67e53e199e1)

**Comment by AlejandroTuzzi** [comment]
No, when you reduce the strength it disappears but also IPAdapter has less influence.

**Comment by al-swaiti** [comment]
mmm did u use x lab one

**Comment by AlejandroTuzzi** [comment]
I tried but is extremely slow to me, didn't happen to you?
